src/ucp/api/ucp.h

```diff
@@ -377,6 +377,20 @@ typedef enum {
 } ucp_atomic_fetch_op_t;
 
 
+/**
+ * @ingroup UCP_COMM
+ * @brief Flags to define behavior of @ref ucp_stream_recv_nb function
+ *
+ * This enumeration defines behavior of @ref ucp_stream_recv_nb function.
+ */
+typedef enum {
+    UCP_RECV_FLAG_WAITALL = UCS_BIT(0)  /**< This flag requests that operation
+                                             will not be completed untill all
```

Comment:
may be rename to UCP_STREAM_RECV_... as this flag is specific for the stream api